code/modules/vehicles/unmanned/unmanned_vehicle.dm

```diff
@@ -14,6 +14,7 @@
 	hud_possible = list(MACHINE_HEALTH_HUD, MACHINE_AMMO_HUD)
 	flags_atom = BUMP_ATTACKABLE
 	soft_armor = list(MELEE = 25, BULLET = 85, LASER = 50, ENERGY = 100, BOMB = 50, BIO = 100, "rad" = 100, FIRE = 25, ACID = 25)
+	var/new_move_delay = 0	//needed to keep track of any slowdowns or diagonal movement
```

Comment:
you should probably rename this to next_move_delay since its more indicative

also missing autodoc